transaction/src/action/ibc.rs

```diff
@@ -31,6 +33,13 @@ impl IsAction for ICS20Withdrawal {
     fn balance_commitment(&self) -> penumbra_crypto::balance::Commitment {
         self.balance().commit(Fr::zero())
     }
+
+    fn decrypt_with_perspective(
+        &self,
+        txp: &TransactionPerspective,
+    ) -> anyhow::Result<Option<ActionView>> {
+        todo!()
```

Comment:
What would this be filled in with, considering that the `ICS20Withdrawal` action doesn't have any shielded data?

Reply:
`None`, maybe, depending on where we fall on what to do when there is nothing to decrypt.

Reply:
This is solved by the `ActionView` design